.gitignore

```diff
@@ -6,7 +6,7 @@ src/main/resources/docs/
 # IDEA files
 /.idea/
 /out/
-*.iml
+/*.iml
```

Comment:
Please revert this change using rebase, the change being committed will override the new change on master.